src/components/Popover/README.md

```diff
@@ -415,6 +417,10 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 Popovers usually contain an [option list](https://polaris.shopify.com/components/lists-and-tables/option-list) or an [action list](https://polaris.shopify.com/components/actions/action-list), but can also contain other controls or content.
 
+A popover can contain many numerous types of content. Whether it's a menu, grid or something entirely different! When `aria-expanded` is applied to an element, `aria-haspopup` will default to `menu`. To assist screen readers you'll need to pass `ariaHaspopup` to `Popover`.
+
+Popovers can contain many types of content. If a popover contains a complex feature such as an ARIA menu, grid, listbox, or similar feature, then use the `ariaHaspopup` prop. This adds `aria-haspopup` to the button, which is helpful information for people who use screen readers.
```

Comment:
Was this section meant to be removed?

Reply:
It was a bad rebase, thanks for catching this 🙏